### ISSUE TITLE
Home location

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#5122]
**[Submitted to the original trac issue database at 4.43pm, Sunday, 16th February 2014]**

When I try to input my "Thuislocatie" ("home location" in English) by clicking on the map in "Mijn instellingen" ("My settings" in English), de cordinates are not accurate.

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 4.49pm, Sunday, 16th February 2014]**

How "not accurate" are we talking? One metre off? A hundred km off?

**Comment by tomhughes** [comment]
**Author: willem.heyvaert@telenet.be**
**[Added to the original trac issue at 4.16pm, Monday, 17th February 2014]**

Replying to [comment:1 TomH]:
Strange. It seems to work right now. I don't know what happened. Yesterday, the difference was about 1 to 3km in Northern direction. Yesterday, I tried to give my home locate on a computer with Chrome installed. Now, I tried to do it at home with Firefox (Chrome is n't installed on my home computer) and it works accurate.


> How "not accurate" are we talking? One metre off? A hundred km off?
